recipes/thermofun/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.1.6" %}
+
+package:
+  name: thermofun
+  version: {{ version }}
+
+source:
+  url: https://github.com/thermomod/thermofun/archive/v{{ version }}.tar.gz
+  sha256: 242e65cfd78cf9dadf6e38c761c86c8dec6f90d0f29482407fc876b65b1c85d2
+
+build:
+  number: 0
+  # skip: True  # [win and vc<15]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake>=3.13
+    - ninja
+    - pybind11>=2.3.0
+    - nlohmann_json>=3.6.1
```

Comment:
pybind and nlohmann_json should be in host

Reply:
Thanks - just changed this as suggested!